tools/RELEASE.md

```diff
@@ -153,8 +155,18 @@ and librdkafka versions:
     $ python -c 'import confluent_kafka as ck ; print "py:", ck.version(), "c:", ck.libversion()'
     py: ('0.11.4', 721920) c: ('0.11.4-RC1', 722121)
 
+## 10. Open a release PR
+
+Once all builds are successful and artifacts have been validated proceed with opening a PR.
```

Comment:
You can do the re-build cycle dance in the PR branch, don't need to wait with creating a PR.